average.js

```diff
@@ -1,6 +1,12 @@
 
 function average(numbers) {
-  return numbers.reduce((p, c)=> p + c, 0) / numbers.length;
+  var noOfNans = 0;
+  return numbers.reduce((total, i) => {
```

Comment:
Nice translation to concise code and good naming of variables as well :+1: 
Minor suggestion: you may want to use another name instead of `i`, since by convention, `i` is used to mean an index.

Reply:
Sure! Will send a PR soon. I just saw this. Something is wrong with my Github notifications. Thanks for reviewing the PR